swarm/pss/pss.go

```diff
@@ -379,7 +379,7 @@ func (p *Pss) handlePssMsg(ctx context.Context, msg interface{}) error {
 	if !ok {
 		return fmt.Errorf("invalid message type. Expected *PssMsg, got %T ", msg)
 	}
-	log.Trace("handler", "self", label(p.Kademlia.BaseAddr()), "topic", label(pssmsg.Payload.Topic[:]))
+	log.Trace("handler", "self, topic", fmt.Sprintf("%s   %s", label(p.Kademlia.BaseAddr()), label(pssmsg.Payload.Topic[:])))
```

Comment:
pourquoi?

Reply:
i fogot to change it back. i did this, because otherwise it produced two lines of code instead of one, and i could not fix it